edge/discussion_materials/ONNX Edge scope and profile definition.md

```diff
@@ -0,0 +1,76 @@
+# ONNX Edge scope and profile definition
+Author: Yedong Liu, Milan Oljaca, Saurabh Tangri, Ofer Rosenberg
+
+## Motivation
+
+The following document provides a description of terminology related to "edge" : what is the scope of "edge", what are the scenarios, example to devices which map into the edge, and more. The goal is to define what is contained inside an edge profile , based on attributes.
+
+The next step would be using these definitions to create a per profile document which contains: the definition of a profile, scenario/s which are relevant to this profile, scenario mapping to list of ONNX models (and hence derived ONNX operators).
+
+## Scope of Edge
+
+Broadly speaking, scope of edge can be defined as "not cloud". Edge computing is computing that’s done at or near the source of the data, without relying on the cloud computing resources in data centers. Edge computation is largely or completely performed on distributed edge devices. The consumer of edge computing is any application or general functionality needing to be closer to the source of the action where system can process and interact with the real world instantly.
+
+## Edge scenario
+
+Doing the computing at the edge may be due to various reasons - need for real-time, privacy, latency, connectivity and more. A few examples for edge usage scenarios there are: face recognition, smart assistant, smart city, intelligent traffic control, industrial IoT, video analysis and monitoring, border control, unattended shops, autonomous cars and smart drones.
```

Comment:
I would recommend remove "border control, unattended shops".
Not sure what "unattended shops" is.